src/code/views/share-dialog-view.tsx

```diff
@@ -130,7 +132,11 @@ export default class ShareDialogView extends React.Component<IShareDialogProps,
     const documentId = encodeURIComponent(this.getShareUrl())
     const separator = this.state.interactiveApiServerUrl.includes('?') ? '&' : '?'
     const togglesParam = this.state.graphVisToggles ? 'app=is&' : ''
-    return `${this.state.interactiveApiServerUrl}${separator}${togglesParam}interactiveApi&documentId=${documentId}`
+    let url = `${this.state.interactiveApiServerUrl}${separator}${togglesParam}interactiveApi&documentId=${documentId}`
+    if (this.state.fullscreenScaling) {
+      url = FULLSCREEN_INTERACTIVE_URL + encodeURIComponent(url)
```

Comment:
If you wrote this as:
```suggestion
      url = ${FULLSCREEN_INTERACTIVE_URL}?wrappedInteractive=${encodeURIComponent(url)}
```
with `FULLSCREEN_INTERACTIVE_URL` defined as `"https://models-resources.concord.org/question-interactives/version/v1.6.0/full-screen/` the intent might be a bit clearer.